pynest/examples/hh_phaseplane.py

```diff
@@ -125,10 +125,10 @@
 print('')
 print('Plot analysis')
 
-V_matrix = [list(x) for x in zip(*V_new_vec)]
-n_matrix = [list(x) for x in zip(*n_new_vec)]
-n_vec = [x/100. for x in range(10, 81)]
-V_vec = [x*1. for x in range(-100, 42, 2)]
+V_matrix = [list(v) for v in zip(*V_new_vec)]
+n_matrix = [list(v) for v in zip(*n_new_vec)]
+n_vec = [v / 100. for v in range(10, 81)]
+V_vec = [v * 1. for v in range(-100, 42, 2)]
```

Comment:
But why does renaming `x` to `v` make a difference?

Reply:
Because `x`, defined here:
https://github.com/nest/nest-simulator/blob/f144599de2345d856a663dc7eb71de4c20862ed3/pynest/examples/hh_phaseplane.py#L62
contains contains phase-plane data as a vector field needed for plotting, which is done here:
https://github.com/nest/nest-simulator/blob/f144599de2345d856a663dc7eb71de4c20862ed3/pynest/examples/hh_phaseplane.py#L149
This is fine in Python 3, as the variable created in list comprehension is kept only within the list comprehension. However, list comprehension in Python 2 overwrites any previously defined  variable.